src/java.base/share/classes/sun/util/cldr/CLDRLocaleProviderAdapter.java

```diff
@@ -224,7 +224,7 @@ protected Set<String> createLanguageTagSet(String category) {
             return Collections.emptySet();
         }
         StringTokenizer tokens = new StringTokenizer(supportedLocaleString);
-        Set<String> tagset = new HashSet<>((tokens.countTokens() * 4 + 2) / 3);
+        Set<String> tagset = HashSet.newHashSet(tokens.countTokens());
```

Comment:
Hi @XenoAmess, during the refactoring of the above `StringTokenizer`, it turned out that this `HashSet` can be replaced with `Set.of()`. So you can leave this piece as it is, as I will take care of it with JDK-8287340.

Reply:
@naotoj done.